ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Config.scala

```diff
@@ -40,21 +36,11 @@ private[perf] object Config {
     Config[String](
       scenario = "",
       dars = List.empty,
-      jwt = Jwt(""),
       reportsDir = new File(""),
       maxDuration = None,
       queryStoreIndex = QueryStoreIndex.No,
     )
 
-  implicit val configInstance: Traverse[Config] = new Traverse[Config] {
```

Comment:
happy this is removed, I don't see the value in a traverse instance on a `Config`?